tools/ci_build/github/linux/docker/inference/aarch64/default/cpu/scripts/install_deps.sh

```diff
@@ -67,3 +67,7 @@ tar --strip 1 -xf /tmp/src/node-v18.17.1-linux-${NODEJS_ARCH}.tar.gz -C /usr
 
 cd /
 rm -rf /tmp/src
+
+git clone https://github.com/Microsoft/vcpkg.git /
```

Comment:
Pin the version. 
You can download the source code via curl and HTTPS as well, to avoid introducing a new dependency, git, in the docker.